setup.py

```diff
@@ -262,6 +262,7 @@ def run(self):
     entry_points={
         'console_scripts': [
             'borg = borg.archiver:main',
+            'borgmount = borg.archiver:main_mount',
```

Comment:
That surprises me.

Well, I still think mount.borg is bad since it breaks out of the "fuse-namespace". None of sshfs, rdiff-backup-fs et all do provide `mount.*`, their binaries are called `sshfs`, `rdiff-backup-fs` and so on. So I guess we should stick to that already somewhat established convention.